# joint_plan.yaml
joint_test_client:
  goal_configuration: [-1.0, 0.0, 0.0, -1.57, 1.3, 1.0, 1.0]
  initial_configuration: [2.0, 1.0, 0.0, -1.27, 1.3, 1.57, 1.0]
  planning_group: "panda_arm"
